### PR TITLE
fix instructions where to find OM version

### PR DESCRIPTION
**Pull request by matkoniecz**
tested for Android - maybe this instructions are correct for Apple devices?

### CODE REVIEW

**Comment by strump** [comment]
This green button does the same as Help section in the settings. Maybe rewrite as "Tap bottom left button with green OrganicMaps logo"?

**Comment by matkoniecz** [comment]
@strump So that is what happened! I now seem to remember this button as having question mark icon.

Description rewritten in seemingly (to me) more readable form, special commit incantation added

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ If applicable, add screenshots or screen recordings to help explain your problem
 
 **System information:**
  - Operating system and its version: [iOS 12, Android 10, Ubuntu 22, MacOS Big Sur, etc.]
- - Organic Maps version: [you can find it by clicking the "?" button]
+ - Organic Maps version: [you can find it by tapping button with green OrganicMaps logo, bottom left of the main screen]
```
Comment (Mr-Mime) [inline .github/ISSUE_TEMPLATE/bug_report.md:27]:
```suggestion
 - Organic Maps version: [you can find it by tapping the button with the green OrganicMaps logo, bottom left of the main screen]
```

I suggest those small grammatical corrections

**Review by Mr-Mime — CHANGES_REQUESTED** [review]

**Comment by biodranik** [comment]
On iOS, there is still a question mark button.

**Comment by matkoniecz** [comment]
So I guess that instructions should mention both? Or is this divergence a bug?

**Comment by biodranik** [comment]
The plan is to update iOS icon too, but until it's not updated, it's better to keep both.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ If applicable, add screenshots or screen recordings to help explain your problem
 
 **System information:**
  - Operating system and its version: [iOS 12, Android 10, Ubuntu 22, MacOS Big Sur, etc.]
- - Organic Maps version: [you can find it by tapping button with green OrganicMaps logo, bottom left of the main screen]
+ - Organic Maps version: [you can find it by tapping the button with the green OrganicMaps logo, bottom left of the main screen]
```
Comment (biodranik) [inline .github/ISSUE_TEMPLATE/bug_report.md:27]:
```suggestion
 - Organic Maps version: [you can find it by tapping the button with the green OrganicMaps logo, bottom left of the main screen on Android, or the `?` button on iOS]
```

**Review by biodranik — APPROVED** [review]
Thanks!

**Pull request merged**